win32_event_log/datadog_checks/win32_event_log/check.py

```diff
@@ -0,0 +1,24 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import AgentCheck, is_affirmative
+
+from .legacy import Win32EventLogWMI
+
+
+class Win32EventLogCheck(AgentCheck):
+    __NAMESPACE__ = 'kafka'
+
+    def __init__(self, name, init_config, instances):
+        super(Win32EventLogCheck, self).__init__(name, init_config, instances)
+
+    def check(self, _):
+        raise NotImplementedError()
+
+    def __new__(cls, name, init_config, instances):
+        instance = instances[0]
+
+        if is_affirmative(instance.get('use_wmi', False)):
```

Comment:
The config should be added to `conf.yaml.example` or it's hidden on purpose?

Reply:
Hidden until I do new implementation